roles/srobo-nginx/templates/nginx.conf

```
@@ -208,6 +208,9 @@ http {
 
     # Volunteering Links
     rewrite ^/join                 https://docs.google.com/forms/d/e/1FAIpQLSdP55Lz_qNEx7t-hdQctt2VzXdYikOe9k2dKsDzEZZG7-nqTw/viewform redirect;
+    
+    # Short link for livestream embed, mostly for use in kiosk screens and other automation
+    rewrite ^/comp-livestream      https://www.youtube-nocookie.com/embed/7HNG2bqwXlo redirect;
```

If this is purely an internal thing, suggest we make this year-specific:
```suggestion
    rewrite ^/sr2022-livestream      https://www.youtube-nocookie.com/embed/7HNG2bqwXlo redirect;
```
If this isn't year specific, would a simpler url be easier?
```suggestion
    rewrite ^/livestream      https://www.youtube-nocookie.com/embed/7HNG2bqwXlo redirect;
```

---

I like the idea of just having a simple URL for it, we can reuse it between events much easier (and non-comp ones).